client/post-editor/editor-sidebar/feedback-list.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import Gravatar from 'components/gravatar';
+
+class FeedbackList extends PureComponent {
+	static propTypes = {
+		sharedLinks: PropTypes.array,
+		onToggleFeedback: PropTypes.func
+	}
+
+	render() {
+		const { sharedLinks } = this.props;
+
+		return (
+			<div className="editor-sidebar__feedback-list">
+				{ sharedLinks.map(
+					sharedLink => this.renderSharedLink( sharedLink )
+				) }
```

Comment:
The purpose of the arrow function was to bind to the context since the rendering helper required it.

I like the visibility and decomposition arguments for not splitting out sub-render functions. I'm fixing the components to respect this now.